WebAPI/LearningHub.Nhs.Services/ResourceService.cs

```diff
@@ -2586,7 +2586,7 @@ public async Task<LearningHubValidationResult> CreateFileDetailsAsync(FileCreate
         {
             if (fileCreateRequestViewModel.ResourceVersionId == 0)
             {
-                fileCreateRequestViewModel.ResourceVersionId = await this.resourceRepository.CreateResourceAsync(fileCreateRequestViewModel.ResourceType, string.Empty, string.Empty, userId);
+                fileCreateRequestViewModel.ResourceVersionId = await this.resourceRepository.CreateResourceAsync(fileCreateRequestViewModel.ResourceType, string.Empty, string.Empty, 1, userId);
```

Comment:
Under what circumstances is this code called? Is setting the PrimaryCatalogueId = 1 correct?